static/css/main.css

```diff
@@ -243,7 +242,7 @@ figure {
 }
 .meta .reading-time::before {
   content: "-";
-  margin-right: 85rem;
+  margin-right: .85rem;
```

Comment:
ahahaha 😆

Reply:
I needed 5 minutes to catch the error ^^